artifactory/1.11/artifactory-pro.md

```diff
@@ -0,0 +1,54 @@
+# Setting up Artifactory Pro on DC/OS
+
+## Prerequisites
+
+- DC/OS 1.11 or later with at least one public agent
+- [DC/OS CLI installed](https://docs.mesosphere.com/1.11/cli/install/) and
+  configured to use your cluster
+- Database (MySQL, Oracle, MS SQL Server or Postgres)
```

Comment:
Do you want to link to artifactory/1.11/install-postgres.md here in case someone missed that step?

Reply:
Sure.